packages/ui/src/components/DataStory/clients/createTransport.ts

```diff
@@ -11,11 +11,15 @@ export const createTransport = (config: TransportConfig): Transport => {
   async function sendAndReceive<T>(params: Record<string, any>): Promise<T> {
     const id = createDataStoryId();
     const message = { msgId: id, ...params };
-    config.postMessage(message);
-    return await firstValueFrom(config.messages$.pipe(
+    const resp = firstValueFrom(config.messages$.pipe(
       filter(it => it.msgId === id),
-      map(it => it as T),
-    ))
+      map(it => {
```

Comment:
pointless map? `it => it` ?

Reply:
Since I often debug here, using regular functions is more convenient. It allows us to add `console` and other logic within the function easily 😂